skopt/optimizer/base.py

```diff
@@ -82,60 +60,39 @@ def gp_minimize(func, dimensions, base_estimator=None, alpha=10e-10,
           `Categorical`).
 
     * `base_estimator` [a Gaussian process estimator]:
-        The Gaussian process estimator to use for optimization.
-        By default, a Matern kernel is used with the following
-        hyperparameters tuned.
-        - All the length scales of the Matern kernel.
-        - The covariance amplitude that each element is multiplied with.
-        - Noise that is added to the matern kernel. The noise is assumed
-          to be iid gaussian.
+        Should be a sklearn Regressor. In addition, should have an optional
+        `return_std` argument, which returns `std(Y | x)`` along with
+        `E[Y | x]`
 
-    * `acq` [string, default=`"EI"`]:
-        Function to minimize over the gaussian prior. Can be either
+    * `n_calls` [int, default=100]:
+        Number of calls to `func`.
+
+    * `n_random_starts` [int, default=10]:
+        Number of evaluations of `func` with random initialization points
+        before approximating the `func` with `base_estimator`.
+
+    * `acq_func` [string, default=`"EI"`]:
+        Function to minimize over the posterior distribution. Can be either
 
         - `"LCB"` for lower confidence bound,
         - `"EI"` for expected improvement,
         - `"PI"` for probability of improvement.
```

Comment:
This way already like this, but I realize we should mention "(negative) expected improvement" and (negative) probability of improvement", since the docstring says we are minimizing those.